client/src/Pages/Overview/OverviewPage.jsx

```diff
@@ -0,0 +1,27 @@
+import Heading from "../../components/Heading";
+import {useAuthContext} from "../../api/useAuthContext";
+import Button from "../../components/Button";
+import {Link} from "react-router-dom";
+
+
+export default function OverviewPage() {
+    const context = useAuthContext();
+    const name = context?.user?.claims?.find(x => x.key === 'name')?.value;
```

Comment:
Burde vi ikke try catch denne ? Hva skjer om denne er feil  eller tom ? Tenker vi kan ha en catch{ name = "namePlaceHolder"} også kan vi lage den bedre senere 👍

Reply:
Dette er en nullable verdi så hvis den ikke finner det så vil den stå som null eller undefined egentlig. Kan smelle på en ternary operator for å håndtere hvis den ikke finner det